xbmc/music/GUIViewStateMusic.cpp

```diff
@@ -188,6 +189,8 @@ CGUIViewStateMusicDatabase::CGUIViewStateMusicDatabase(const CFileItemList& item
       AddSortMethod(SortByArtist, sortAttribute, 557, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));  // Filename, empty | Userdefined, Userdefined
       // year
       AddSortMethod(SortByYear, 562, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));
+      // album date added
+      AddSortMethod(SortByDateAdded, sortAttribute, 570, LABEL_MASKS("%F", "", strAlbumLeft + " - " + strAlbumRight, "%a"));  // Filename, empty | Userdefined - Userdefined, dateAdded
```

Comment:
I would do it the same as the existing sort methods for now and then maybe make a PR to adjust them all and see what the feedback is.